services/web/pkg/apps/apps.go

```diff
@@ -129,6 +123,7 @@ func Build(fSystem fs.FS, id string, conf map[string]any) (Application, error) {
 		return Application{}, errors.Join(err, ErrInvalidManifest)
 	}
 
+	validate := validator.New(validator.WithRequiredStructEnabled())
```

Comment:
can you explain why we do not have it global anymore? not a fan to re-initialize it on every build call

Reply:
Init function is discouraged.
Is the validator.new expensive?